### PR TITLE
Park 5 Create Zones

### PR DESCRIPTION
**Pull request by marras0914**

### CODE REVIEW

File: src/manage/CreateZone/CreateZone.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    font-family: Muli;
+    font-style: normal;
+    font-weight: bold;
+    font-size 18px;
+    color: #4D4D4D;
+    width: 100%;
+    margin: 0 0 .5rem 0;
+  }
+ 
+  button {
```
Comment (aVileBroker) [inline src/manage/CreateZone/CreateZone.jsx:44]: Why not just make these styled components?

File: src/manage/CreateZone/CreateZone.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  justify-content: space-between;
+  height: 100%;
+  h2 {
+    margin: 0;
+    line-height: 40px;
+    font-family: Muli;
+    font-style: normal;
+    font-weight: bold;
+    font-size 30px;
+    color: #4D4D4D;
```
Comment (aVileBroker) [inline src/manage/CreateZone/CreateZone.jsx:23]: at some point we should pull these into a constant file - but not a blocker for this PR

File: src/manage/CreateZone/CreateZone.jsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+`;
+
+const StyledButton = styled(Button)`
+  background-color: #8D1EF8;
+`;
+
+// purple: #8D1EF8;
+export default ({createZone}) => {
+  const dispatch = useDispatch();
+  const handleDelete = (boundary) => {
```
Comment (aVileBroker) [inline src/manage/CreateZone/CreateZone.jsx:90]: should this function be called handleToggle?

File: src/manage/CreateZone/CreateZone.jsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+            label={boundary}
+            onDelete={() => handleDelete(boundary)}
+            deleteIcon={<Icon size="1rem" path={mdiClose} />}>
+          </Chip> 
+        ))}
+      </StyledBoundriesContainer>
+   
+      <StyledCard>
+        <h3>Editors</h3>
+        <StyledLabel htmlFor="">Can do everything except edit zone boundaries.</StyledLabel>
```
Comment (aVileBroker) [inline src/manage/CreateZone/CreateZone.jsx:135]: should `htmlFor` be `editorEmail`?

File: src/manage/CreateZone/CreateZone.jsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        <TextField id="editorEmail" size="small" variant="outlined" type="email"/>
+        <button>
+          <Icon size="1rem" path={mdiPlus} />
+          Add an Editor
+        </button>
+      </StyledCard>
+      <StyledCard>
+        <h3>Viewers</h3>
+        <StyledLabel htmlFor="">Can do everything except edit zone boundaries.</StyledLabel>
```
Comment (aVileBroker) [inline src/manage/CreateZone/CreateZone.jsx:144]: same htmlFor question

File: src/manage/CreateZone/LeafletMap.jsx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  }
+};
+
+const StyledMap = styled(Map)`
+  // position: fixed;
+  // top: 0;
+  // left: 0;
+  height: 100%;
+  width: 67vw;
+  // z-index: -1;
```
Comment (aVileBroker) [inline src/manage/CreateZone/LeafletMap.jsx:40]: let's just delete these comments if we don't need them

File: src/manage/CreateZone/index.jsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import LeafletMap from './LeafletMap';
+import CreateZone from './CreateZone';
+import styled from 'styled-components';
+
+
+const pittsburghDistricts = require('../../mockData/Pittsburgh_City_Council_Districts_2012.json')
```
Comment (aVileBroker) [inline src/manage/CreateZone/index.jsx:7]: I think we can just do `import pittsburghDistricts from '../../mockData/Pitts...`

File: package.json [hunk 8]
```diff
@@ -3,14 +3,19 @@
...
+    "@mdi/js": "^5.2.45",
+    "@mdi/react": "^1.4.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "@turf/bearing": "^6.0.1",
     "@turf/destination": "^6.0.1",
     "@turf/distance": "^6.0.1",
     "eslint": "^6.8.0",
+    "install": "^0.13.0",
```
[the remark this answers is not in the document]
Comment (marras0914) [reply package.json:16]: that was an accident

File: package.json [hunk 9]
```diff
@@ -3,14 +3,19 @@
...
     "leaflet": "^1.6.0",
+    "npm": "^6.14.5",
```
[the remark this answers is not in the document]
Comment (marras0914) [reply package.json:18]: also an accident, its what happens when you do yarn add npm install ...

File: src/containers/CreateZoneContainer.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+const mapStateToProps = (state) => {
+  return {
+    createZone: state.createZone
+  }
+}
+
+export default connect(
+  mapStateToProps,
+  null
+)(CreateZoneContainer);
```
[the remark this answers is not in the document]
Comment (marras0914) [reply src/containers/CreateZoneContainer.js:26]: no

File: src/manage/CreateZone/CreateZone.jsx (hunk 3 above)
Comment (marras0914) [reply src/manage/CreateZone/CreateZone.jsx:90]: It only goes one way. It only removes them.

File: src/manage/CreateZone/index.jsx [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+  position: relative;
+  display: flex;  
+  flex-flow: row wrap;
+  width: 100vw;
+  height: calc(100vh - 5rem);
+  top: 5rem;
+`;
+export default () => {
+
+  return (
```
Comment (aVileBroker) [inline src/manage/CreateZone/index.jsx:19]: can be condensed (and should throw a lint error I believe?)

**Pull request merged**